docker-compose.yml

```diff
@@ -12,6 +12,12 @@ volumes:
   manager-data:
   postgresql-data:
 
+x-logging: &default-logging
+  driver: syslog
+  options:
+    syslog-address: "tcp://manager:1514"
```

Comment:
Does this driver use the host network rather than the docker network created for the stack? If so would need to use localhost and map port `127.0.01:1514` of the manager container

Reply:
The driver does use the host network, but there is a new mode called "non-blocking", which means that the docker stack will start regardless of the driver. This means that we could boot up the stack and then receive the syslogs. We could probably use `localhost` and only bind that port of the manager to `127.0.0.1`, so that it's not available outside the host. Again, eventually, it would be cool if OpenRemote could handle outside syslog requests (maybe let's say a remote device?) but that's out of the scope of this PR for now.